packages/amplify_api/android/build.gradle

```diff
@@ -55,8 +55,8 @@ dependencies {
     api amplifyCore
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation "com.amplifyframework:aws-api:1.17.3"
-    implementation "com.amplifyframework:aws-api-appsync:1.17.3"
+    implementation "com.amplifyframework:aws-api:1.17.4"
+    implementation "com.amplifyframework:aws-api-appsync:1.17.4"
```

Comment:
Nit: Inconsistent use of single and double quotes across the `build.gradle` files

Reply:
We can fix this another time.